server/src/main/java/com/cloud/network/NetworkModelImpl.java

```diff
@@ -2207,14 +2207,8 @@ public boolean isNetworkInlineMode(Network network) {
 
     @Override
     public void checkIp6Parameters(String startIPv6, String endIPv6, String ip6Gateway, String ip6Cidr) throws InvalidParameterValueException {
-        if (!NetUtils.isValidIp6(startIPv6)) {
-            throw new InvalidParameterValueException("Invalid format for the startIPv6 parameter");
-        }
-        if (!NetUtils.isValidIp6(endIPv6)) {
-            throw new InvalidParameterValueException("Invalid format for the endIPv6 parameter");
-        }
 
-        if (!(ip6Gateway != null && ip6Cidr != null)) {
+        if (StringUtils.isBlank(ip6Gateway) || StringUtils.isBlank(ip6Cidr)) {
             throw new InvalidParameterValueException("ip6Gateway and ip6Cidr should be defined when startIPv6/endIPv6 are passed in");
```

Comment:
@GabrielBrascher may be this exception msg has to be updated, as it is not sure here that startIPv6/endIPv6 are passed or not, as the checks for startIPv6/endIPv6 are moved after this.

Reply:
Just pushed a commit addressing your comment. Thanks Suresh.